source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,361 @@
+Up to this point, we've generated four top level routes.
+
+* An `about` route, that gives information on our application.
+* A `contact` route, with information on how to contact the company.
+* A `rentals` route, where we will allow users to browse rental properties.
+* and the `index` route, which we've set up to redirect to the `rentals` route.
```

Comment:
Should this `and` be capitalized? Or perhaps use a comma instead of a period on the line above?